### ISSUE TITLE
If there is Github repo connected Issues section with the list of current issues open

### ISSUE THREAD

**Issue by evshvarov** [intersystems-community/openexchange#55]

**Comment by gnibeda** [comment]
Acceptance criteria:
* There is new tab "Issues" on package details screen
* Tab visible only if github repo was set and there are some issues on github repo
* Issues are to be loaded only after tab was clicked
* There is loader spinner on "Issues" tab during loading
* There is new button "Add issue" on issues tab
* Clicking on "Add issue" button will open new tab with github new issue page

**Comment by MakarovS96** [comment]
done
